src/zcbenchmarks.cpp

```diff
@@ -116,10 +117,24 @@ double benchmark_solve_equihash()
                                     nonce.begin(),
                                     nonce.size());
 
-    timer_start();
+    if (time)
+        timer_start();
```

Comment:
`timer_start()` changes a file-global variable, so the threads would interfere with the outer timing. Also, we don't need the (admittedly minimal) overhead of the function call inside each thread.